aws_eventbridge/eventbridge-single-region.yaml

```diff
@@ -0,0 +1,125 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Datadog AWS Cloudtrail Eventbright Logs Integration
+Parameters:
+  ApiKey:
+    Description: >-
+      Your Datadog API Key
+    Type: String
+    AllowedPattern: .+
+    ConstraintDescription: ApiKey is required
+  DatadogCloudtrailServiceRoleArn:
+    Description: >-
+      The arn for the service role used by kinesis firehose
+    Type: String
+    AllowedPattern: .+
+    ConstraintDescription: DatadogCloudtrailServiceRoleArn is required
+  DatadogCloudtrailEventbridgeRoleArn:
+    Description: >-
+      The arn for the eventbridge used by the eventbridge
+    Type: String
+    AllowedPattern: .+
+    ConstraintDescription: StreamRoleArn is required
+  DdSite:
+    Description: >-
+      Define your Datadog Site to send data to. For the Datadog EU site, set to datadoghq.eu
+    Type: String
+    AllowedPattern: .+
+    Default: 'datadoghq.com'
+    ConstraintDescription: DdSite is required
+Conditions:
+  EUDatacenter: !Equals [ !Ref DdSite, 'datadoghq.eu' ]
+  US5Datacenter: !Equals [ !Ref DdSite, 'us5.datadoghq.com' ]
+  US3Datacenter: !Equals [ !Ref DdSite, 'us3.datadoghq.com' ]
+  Staging: !Equals [ !Ref DdSite, 'datad0g.com' ]
+Resources:
+  DatadogCloudtrailLogs:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: "datadog-eventbridge-cloudtrail-stream"
+      RetentionInDays: 14
+  HTTPLogStream:
+    Type: AWS::Logs::LogStream
+    Properties:
+      LogGroupName: !Ref DatadogCloudtrailLogs
+      LogStreamName: "http_endpoint_delivery"
+  S3Backup:
+    Type: AWS::Logs::LogStream
+    Properties:
+      LogGroupName: !Ref DatadogCloudtrailLogs
+      LogStreamName: "s3_backup"
+  DatadogCloudtrailBackupBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !Sub "datadog-aws-cloudtrail-backup-${AWS::AccountId}-${AWS::Region}"
+  DatadogCloudtrailKinesisFirehose:
+    Type: AWS::KinesisFirehose::DeliveryStream
+    Properties:
+      DeliveryStreamName: "datadog-eventbridge-cloudtrail-stream"
+      DeliveryStreamType: "DirectPut"
+      HttpEndpointDestinationConfiguration:
+        BufferingHints:
+          SizeInMBs: 4
+          IntervalInSeconds: 60
+        EndpointConfiguration:
+          Url:
```

Comment:
Is there a reason this is not using a string replace using `DDSite`?  You should be able to drop the conditionals and all these ifs.  Something like this:
```
Url: !Sub ""https://aws-kinesis-http-intake.logs.${DDSite}/v1/input"
```

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html